app/models/verse.rb

```diff
@@ -13,11 +13,22 @@ def self.last_month_winner
 
   def self.previous_month
     # 1.month.ago
-    current_month = Time.now.mon
     if current_month == 1
       return 12
     else
       current_month - 1
     end
   end
+
+  def self.current_month
+    current_month = Time.now.mon
+  end
+
+  def self.top_three
+    competition = Competition.find_by(month: current_month)
+    contenders = competition.verses.sort_by do |verse|
+      verse.votes.count
+    end.reverse
+    contenders[0..2]
```

Comment:
This looks good, though I think we want to refactor this into ActiveRecord at some point for teh performance;  made an issue as a reminder

Reply:
Good call Eugene - an order method here should do the trick